.travis.yml

```diff
@@ -1,8 +1,9 @@
 sudo: false
 language: node_js
 node_js:
-  - "4"
   - node
+  - 6
+  - 4
```

Comment:
If you are going to validate against multiple version, you should do it in a matrix and use the whole `JOB_PART` setup from webpack/webpack. 

Right now if you add more validation tasks ( and they are coming ) they will execute against every version which is unnecessary & inconsiderate to Travis who is running all this for free :)